com.unity.render-pipelines.universal/Shaders/Nature/SpeedTree8Passes.hlsl

```diff
@@ -267,6 +268,9 @@ SpeedTreeVertexOutput SpeedTree8Vert(SpeedTreeVertexInput input)
 
     output.clipPos = vertexInput.positionCS;
 
+    OUTPUT_LIGHTMAP_UV(input.texcoord1.xy, unity_LightmapST, output.lightmapUV);
```

Comment:
OUTPUT_LIGHTMAP_UV(input.texcoord1.xy, unity_LightmapST, output.lightmapUV); should not be needed if you have removed all traces of lightmapping. SH below should suffice.